packages/react-native-reanimated/RNReanimated.podspec

```diff
@@ -83,62 +83,21 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/software-mansion/react-native-reanimated.git", :tag => "#{s.version}" }
 
   s.subspec "reanimated" do |ss|
+    ss.source_files = "Common/cpp/reanimated/**/*.{cpp,h}"
+    ss.header_dir = "reanimated"
+    ss.header_mappings_dir = "Common/cpp/reanimated"
```

Comment:
`header_mappings_dir` prevents flattening of header paths, i.e. if the file is located in `x/y/z/foo.h`, it would be available in `reanimated/foo.h` but with `header_mappings_dir` set it will be in `reanimated/x/y/z/foo.h`.